doc/Language/control.pod6

```diff
@@ -927,6 +951,26 @@ for @x -> $x {
 
 prints "12".
 
+If the L<C<LAST> phaser|/language/phasers#LAST> is present, it runs
+before exiting the loop:
```

Comment:
Still not sure about the placement of these examples, within the section that talks about the command with the same (lowercase) name. Is there some reason you haven't placed them within the new `Phasers` section you have created?

Reply:
Yes, there is a reason: next/last are closely related to NEXT/LAST, and should be mentioned together. Without mentioning NEXT/LAST in the same place the reader may think that they always go to the next iteration or exit the loop immediately, and this is not the case. It's also similar to how "return" is already documented in this same document: "The sub return will stop execution of a subroutine or method, run all relevant phasers and provide the given return value to the caller."